screens/RequestScreen.js

```diff
@@ -0,0 +1,100 @@
+import * as WebBrowser from 'expo-web-browser';
+import * as React from 'react';
+import { Component } from 'react';
+import {
+  Picker,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+  Button,
+} from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
+export class RequestScreen extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={{ color: '#fff', fontSize: 24, fontWeight: 'bold' }}>
+          What item are you looking for?
```

Comment:
Maybe we should change to "What item would you like to request?" to differentiate this more from the form on the HomeScreen